src/components/views/Agreement.js

```diff
@@ -200,6 +200,16 @@ class Agreement extends React.Component {
     };
   }
 
+  showTermsAccordion = () => {
+    const { data } = this.props;
+    const primaryprops = data.supplementaryProperties?.filter(p => p.primary === true).length;
+    let custprops = false;
```

Comment:
We should probably initialise this to 0 to stay consistent with this variable holding a length later on.